app/components/patients/AddPatientContainer.js

```diff
@@ -68,7 +68,7 @@ const AddPatientDecorator = AddPatient => {
   }
 }
 
-const mapStateToProps = ({ user }) => ({ user })
+const mapStateToProps = ({ user }) => ({ user });
```

Comment:
There was an issue where the currentPatient wasn't being added and not being passed to my newPlan component so I was using it as a reference.  So I must have placed semi-colons while I was reading through it, but did not make any changes to add patient besides that.